src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -102,6 +103,10 @@ public void onEnable() {
 
         kitchen = new Kitchen(this, miscCategory);
         kitchen.register(this);
+
+		Research kitchenResearch = new Research(new NamespacedKey(this, "kitchen"), 600, "Kitchen", 30);
+		kitchenResearch.addItems(kitchen);
+		kitchenResearch.register();
         
         Research kitchenResearch = new Research(new NamespacedKey(this, "kitchen"), 600, "Kitchen", 30);
```

Comment:
This is a duplicate which results in this not being able to compile successfully.